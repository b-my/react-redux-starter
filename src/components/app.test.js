import React from 'react';
import {shallow} from 'enzyme';
import App from './app';

describe('App', () => {
  test('It shows the correct text', () => {
    const component = shallow(<App />);
    expect(component.find('div').text()).toEqual('React redux starter');
    expect(component).toMatchSnapshot();
  });
});