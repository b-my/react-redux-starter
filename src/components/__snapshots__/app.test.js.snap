// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App It shows the correct text 1`] = `
<div>
  React redux starter
</div>
`;
